subtests/example/example.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+#
+# FIXME - add a one-line description of what this script is supposed to test
```

Comment:
How about:

```
# FIXME - add a one-line description of what this script is supposed to test
# then go to near the bottom of this file to write you test.
```
or something to that effect.  Prevents needing to read the entire script to understand what needs to be done, and where.